en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+
+You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+
+#### Examples
+
+- Multiplying two integer literals
+  ```
+  10 * 20
+  ```
+- Multiplying two decimal literals
+  ```
+  0.56 * 1.84
+  ```
+- Multiplying a literal and a variable
+  ```
+  100 * nameVariable
+  ```
+- Multiplying two variables
+  ```
+  firstNumber * secondNumber
+  ```
+
+### Modulo Operation
+
+You can use the  `%` operator to get the remainder of a division. The number values can come from a literal or a variable.
+
+#### Examples
+- Modulus of two integer literals
+  ```
+  10 % 4
+  ```
+- Modulus of a literal and a variable
+  ```
+  100 % nameVariable
+  ```
+- Modulus of two variables
+  ```
+  firstNumber % secondNumber
+  ```
+
+### Checking The Equality of Basic Types 
+
+In the Ballerina language, basic types are strings, integers, decimal numbers, and booleans. You can compare the values of these types using the `==` binary operator. The values can come from a literal or a variable. Similarly `!=` operator is used to check the inequality. Please note that the resulting value is always of type boolean from these operators.
+
+#### Examples
+- Equality of two integer literals
+  ```
+  10 == 4
+  ```
+- Equality of two string literals
+  ```
+  "hi" == "hi"
+  ```
+- Equality of a literal and a variable
+  ```
+  100 == expectedValueVariable
+  ```
+- Equality of two variables
+  ```
+  firstNumber == secondNumber
+  ```
+- Inequality of two variables. The following will return true when the value of two variables are different.
+  ```
+  firstNumber != secondNumber
+  ```
+
+### Checking the Type of Variable
+
+The `is` operator is used to assert the type of a variable.
+
+#### Examples
+- Checking whether the variable is of type JSON
+  ```ballerina
+  payload is JSON
+  ```
+
+### Numerical Value Comparisons
+
+The following operators are available in the expression editor for numeric value comparisons
+- `<` (less than)
+- `>` (greater than)
+- `<=` (less than or equal to)
+- `>=` (greater than or equal to)
+
+#### Examples
+- Check if the value of a variable is greater than 10
+  ```
+  myVar > 10
+  ```
+- Comparing two variables
+  ```
+  myProperty <= limit
+  ```
+
+## HTTP request related operations
+
+All the HTTP-related service applications created in Choreo have a variable called `req` (of type `http:Request`) which the developers can use to query information related to the HTTP request. Following is a list of use cases that an integration application developer would need more likely need to develop Choreo applications
+
+### Reading a header value in the request
+A typical HTTP request contains many headers. Some of these headers are standard, and some are custom. In an expression-enabled input field, you can use the `req` variables `getHeader` function to read a specific header value.
+
+#### Examples
+- Reading the value of Access-Control-Allow-Origin HTTP header
```

Comment:
Some bullet points under "Examples" have a punctuation mark at the end of the sentence (e.g. the next bullet point has a period because it uses two sentences), whereas some do not. Let's try to follow a standard?